### PR TITLE
Remove old translation system and add new languages

### PR DESCRIPTION
**Pull request by dospuntos**
This PR removes the original translation system (`LanguageRoster.*` etc.), now that Haiku localization has been implemented in a recent PR.

A translatable string has been added for the location of help files, which are located inside `helpfiles\<Language>\`. Currently available in English, Spanish and Norwegian.

Catkeys included in English, Spanish and Norwegian, fully translated. (The Spanish translations are mostly copied from the orignal translation file, with some minor modifications).

### CODE REVIEW

File: src/ReportWindow.cpp [hunk 1]
```diff
@@ -140,7 +139,7 @@ ReportWindow::ReportWindow(BRect frame)
 	prefsLock.Lock();
 	BString reporthelp = gAppPath;
 	prefsLock.Unlock();
-	reporthelp << "helpfiles/" << gCurrentLanguage->Name() << "/Report Window Help";
+	reporthelp << "helpfiles/" << B_TRANSLATE_COMMENT("English","Path to localized helpfiles. Only translate if available in your language.") << "/Report Window Help";
```
Comment (korli) [inline src/ReportWindow.cpp:142]: @pulkomandy is this correct? It's a bit abused to know if the helpfiles are present in the current language.

Comment (pulkomandy) [reply src/ReportWindow.cpp:142]: This allows to have languages with translations for the application, but not for the documentation, and fall back to English.

I don't see another way to do that without hardcoding the information in the sourcecode.

Comment (korli) [reply src/ReportWindow.cpp:142]: It could also checks at runtime, but more changes are then needed.

**Comment by korli** [comment]
Thanks!

**Pull request merged**